content/telegraf/v1/install.md

```diff
@@ -203,79 +159,88 @@ It can be installed by running:
 sudo pkg install telegraf
 ```
 
-The configuration file is located at `/usr/local/etc/telegraf.conf` with examples in `/usr/local/etc/telegraf.conf.sample`.
-
-{{% telegraf/verify %}}
-
-## Configuration
-
-### Create a configuration file with default input and output plugins.
-
-Every plugin will be in the file, but most will be commented out.
-
-```
-telegraf config > telegraf.conf
-```
-
-### Create a configuration file with specific inputs and outputs
-```
-telegraf --input-filter <pluginname>[:<pluginname>] --output-filter <outputname>[:<outputname>] config > telegraf.conf
-```
+The `telegraf` configuration file is installed at `/usr/local/etc/telegraf.conf`.
+Examples are installed at `/usr/local/etc/telegraf.conf.sample`.
 
-For more advanced configuration details, see the
-[configuration documentation](/telegraf/v1/administration/configuration/).
 {{% /tab-content %}}
-<!---------- BEGIN macOS ---------->
+<!---------- BEGIN MACOS ---------->
 {{% tab-content %}}
 Users of macOS 10.8 and higher can install Telegraf using the [Homebrew](http://brew.sh/) package manager.
-Once `brew` is installed, you can install Telegraf by running:
 
-```bash
-brew update
-brew install telegraf
-```
+1.  If you haven't already, follow the instructions to install the [Homebrew](http://brew.sh/) package manager.
```

Comment:
I will point out that the homebrew package is not a static binary and not the same binary that we provide on the downloads page. This is fine, but it does mean that users find out when something goes wrong.

In practice other than the homebrew binary being static, it does mean that the CPU plugin for example works since it needs CGO support, while the binary we build does not support the CPU plugin.